.github/workflows/test-nightly.yml

```diff
@@ -10,6 +10,12 @@ jobs:
       matrix:
         operating-system: [ubuntu-latest, macOS-latest, windows-latest]
         java-version: [8, 11, 14]
+        archtecture: ['x64', 'x32']
```

Comment:
arch_i_tecture

Reply:
@ibauersachs Thanks a lot! Fixed it in 2d2c40de6fb6af3db2a3d2bedd57eb02442521c4.